source/write-operations/replace-documents.txt

```diff
@@ -0,0 +1,212 @@
+         MongoDB Server manual.  
+
+   * - ``comment(String comment)``
+     - | Attaches a ``String`` comment to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         MongoDB Server manual.  
+
+   * - ``hint(Bson hint)``
+     - | Sets the index for the operation as a ``Bson`` value. 
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the MongoDB Server manual.
+
+   * - ``hintString(String hint)``
+     - | Sets the index for the operation as a ``String`` value.
+         For more information, see the :manual:`hint statement </reference/command/update/#std-label-update-command-hint>`
+         in the MongoDB Server manual.
+
+   * - ``let(Bson variables)``
+     - | Specifies a map of parameter names and values. Values must be constant or closed
+         expressions that don't reference document fields. For more information,
+         see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         MongoDB Server manual.
+
+   * - ``upsert(Boolean upsert)``
+     - | Specifies whether the replace operation performs an upsert operation if no 
+         documents match the query filter. For more information, see the :manual:`upsert
+         statement </reference/command/update/#std-label-update-command-upsert>`
+         in the MongoDB Server manual.
+
+Example
+~~~~~~~
+
+The following code uses the ``replaceOne()`` method to replace a document in the
+``restaurants`` collection. It also sets the ``upsert(true)`` option
+to bypass document-level validation. Because the ``upsert`` option is
+set to ``true``, the driver inserts a new document if the query filter doesn't 
+match any existing documents.
+
+.. code-block:: java
+
+    Mono.from(restaurants.replaceOne(
+      eq("name", "Food Town"),
+      new Document().append("name", "Food World")
+      .append("cuisine", "Mixed"),
+      new ReplaceOptions().upsert(true))).block();
+
+Return Value
+~~~~~~~~~~~~
+
+The ``replaceOne()`` method returns an ``UpdateResult`` 
+object. The ``UpdateResult`` type contains the following properties:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Property
+     - Description
+
+   * - ``getMatchedCount()``
+     - | The number of documents that matched the query filter, regardless of
+         how many were replaced.
+
+   * - ``getModifiedCount()``
+     - | The number of documents modified by the replace operation. If a replaced
+         document is identical to the original, it is not included in this
+         count.
+
+   * - ``getUpsertedId()``
+     - | The ID of the document that was upserted in the database, if the driver
+         performed an upsert. Otherwise ``null``.
+         
+   * - ``wasAcknowledged()``
+     - | Returns true if the replacement was acknowledged.
```

Comment:
I: These are presented as methods based on the parentheses
S: remove parentheses. Also, make the descriptions all the same part of speech

```suggestion
   * - ``getMatchedCount()``
     - | The number of documents that matched the query filter, regardless of
         how many were replaced.

   * - ``getModifiedCount()``
     - | The number of documents modified by the replace operation. If a replaced
         document is identical to the original, it is not included in this
         count.

   * - ``getUpsertedId()``
     - | The ID of the document that was inserted into the database, if the driver
         performed an upsert. If no document was inserted, this value is ``null``.
         
   * - ``wasAcknowledged()``
     - | An acknowledgment of the replacement.
```

Reply:
To my knowledge, these are methods: https://mongodb.github.io/mongo-java-driver/5.1/apidocs/mongodb-driver-core/com/mongodb/client/result/DeleteResult.html#getDeletedCount()

I kept the parentheses but changed the descriptions.